extensions/bitwarden/src/search.tsx

```diff
@@ -54,16 +54,18 @@ export default function Search() {
 export function ItemList(props: { api: Bitwarden }) {
   const bitwardenApi = props.api;
   const session = useSession();
-  const [state, setState] = useState<{ items: Item[]; isLocked: boolean; isLoading: boolean }>({
+  const [state, setState] = useState<{ items: Item[]; folders: Folder[]; isLocked: boolean; isLoading: boolean }>({
     items: [],
+    folders: [],
     isLocked: false,
     isLoading: true,
   });
 
   async function loadItems(sessionToken: string) {
     try {
+      const folders = await bitwardenApi.listFolders(sessionToken);
       const items = await bitwardenApi.listItems(sessionToken);
```

Comment:
We should make those two requests in parallel here.
The startup time of the bitwarden extension is already pretty slow, so the last thing we want is too increase it.